src/overlays/actors/ovl_En_Hgo/z_en_hgo.c

```diff
@@ -28,54 +52,340 @@ const ActorInit En_Hgo_InitVars = {
     (ActorFunc)EnHgo_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BD0F48 = {
-    { COLTYPE_NONE, AT_NONE, AC_NONE, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_2, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
+static ActorAnimationEntry sAnimations[] = {
+    { &D_0600B644, 1.0f, 0.0f, 0.0f, 0, -4.0f }, { &D_06013684, 1.0f, 0.0f, 0.0f, 0, 0.0f },
+    { &D_060152EC, 1.0f, 0.0f, 0.0f, 2, 0.0f },  { &D_06015C70, 1.0f, 0.0f, 0.0f, 0, 0.0f },
+    { &D_060165F0, 1.0f, 0.0f, 0.0f, 0, 0.0f },  { &D_06014220, 1.0f, 0.0f, 0.0f, 2, 0.0f },
+    { &D_06014A9C, 1.0f, 0.0f, 0.0f, 0, 0.0f },
+};
+
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_NONE,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_2,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x00000000, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_NONE,
+        OCELEM_ON,
+    },
     { 18, 46, 0, { 0, 0, 0 } },
 };
 
-// sColChkInfoInit
-static CollisionCheckInfoInit2 D_80BD0F74 = { 0, 0, 0, 0, MASS_IMMOVABLE };
+static CollisionCheckInfoInit2 sColChkInfoInit = { 0, 0, 0, 0, MASS_IMMOVABLE };
+
+// Eye Textures
+static TexturePtr* sEyeTextures[] = {
+    &D_06011138,
+    &D_06011938,
+    &D_06012138,
+};
```

Comment:
```suggestion
static TexturePtr sEyeTextures[] = {
    D_06011138,
    D_06011938,
    D_06012138,
};
```
Don't need the `*` at the end of `TexturePtr` or the `&`s.

Reply:
Good point. I was getting things in a working state and planned on documenting more of the actor, but I don't have most of the analysis tools on my Mac, so I left the symbols in there as a reminder to myself. Perhaps I'll draft this PR in the mean time.